Lib/gftools/encodings/GF Glyph Sets/GF-latin-core_unique-glyphs.nam

```diff
@@ -123,9 +118,6 @@
 0x00B9 ¹ SUPERSCRIPT ONE
 0x00BA º MASCULINE ORDINAL INDICATOR
 0x00BB » RIGHT-POINTING DOUBLE ANGLE QUOTATION MARK
-0x00BC ¼ VULGAR FRACTION ONE QUARTER
-0x00BD ½ VULGAR FRACTION ONE HALF
-0x00BE ¾ VULGAR FRACTION THREE QUARTERS
```

Comment:
I'd honestly keep these fractions. I know this is anecdotal but I use these fractions frequently and would be annoyed if a font didn't include them.

Reply:
We've talked about moving the fractions in the GF Plus encoding which is currently the basic set for commissioned fonts. 
Latin Core scope/goal would be to include the essential to support "most common" Latin languages as a basic ground for new fonts (particularly for not commissioned ones).

Reply:
It would take us back to one of the original concerns about including (or not) the `fraction` as part of the core set. Original reasoning around that was due to bad designed/spaced fractions as well as missing OT feature for them.